docs/source/guide/explanation/algorithms/anomaly/index.rst

```diff
@@ -1,10 +1,123 @@
 
+This section examines the solutions for anomaly detection offered by the OpenVINO Training Extensions library.
 
-   anomaly_classification
-   anomaly_detection
-   anomaly_segmentation
+
+Task Types
+**********
+OTX supports the following anomaly task types:
+
+* Anomaly Classification - (Image-level detection)
+* Anomaly Detection - (Box-level localization)
+* Anomaly Segmentation - (Pixel-level localization)
+
+.. note::
+   Only normal images are used to train all anomaly detection algorithms. The distinction between the tasks is the way in which the testing phase predicts or localises anomalies. In anomaly classification, for instance, anomalies are predicted at the image level; in anomaly detection, anomalies are localised using a bounding box; and in anomaly segmentation, anomalies are localised using a mask.
+
+
+Anomaly Classification
+----------------------
+Anomaly classification is the task of predicting normal and abnormal images at the image level. As noted above, a model is trained on only normal images. During the testing phase, the model predicts an anomaly score indicating the likelihood of an image being abnormal. The threshold for anomaly classification is either set by the user or adaptively tuned by OTX. An image is classified as abnormal if the anomaly score is above the threshold.
+
+Anomaly Detection
+-----------------
+Anomaly detection is the task of predicting normal and abnormal images at box level. Similar to anomaly classification, a model is trained on normal images only. During the testing phase, the model outputs an anomaly heatmap showing the likelihood of each pixel being abnormal. After post-processing the heatmap, the model predicts a bounding box around the anomaly.
+
+Anomaly Segmentation
+--------------------
+The anomaly segmentation task locates anomalies at the pixel level. Similar to anomaly classification and detection, a model is trained on only normal images. During the validation and testing phase, the model outputs an anomaly heatmap showing the likelihood of each pixel being abnormal. After post-processing the heatmap, the model predicts a mask around the anomaly.
+
+
+.. _fig-anomaly-tasks:
+
+.. figure:: ../../../../../utils/images/anomaly_tasks.png
+   :width: 600
+   :align: center
+   :alt: Anomaly Task Types
+
+   Anomaly task types (a) A normal image used during training. (b) An image-level prediction of an anomaly by anomaly classification task (c) A box-level prediction of an anomaly by an anomaly detection task (d) A pixel-level prediction of an anomaly by anomaly segmentation task
+
```

Comment:
We added the descriptions of the training parameters for the individual models. (except for PADIM, because its training does not involve fine-tuning of network weights).